common/api/core-geometry.api.md

```diff
@@ -4534,6 +4581,7 @@ export class PolyfaceQuery {
     static buildPerFaceNormals(polyface: IndexedPolyface): void;
     static cloneByFacetDuplication(source: Polyface, includeSingletons: boolean, clusterSelector: DuplicateFacetClusterSelector): Polyface;
     static cloneFiltered(source: Polyface | PolyfaceVisitor, filter: (visitor: PolyfaceVisitor) => boolean): Polyface;
+    static cloneOffset(source: IndexedPolyface, signedOffsetDistance: number, offsetOptions?: OffsetMeshOptions): IndexedPolyface;
```

Comment:
Could you add a description of this new API to docs/changehistory/NextVersion.md? Or would you prefer I do it?

Reply:
(Deferring to EDL, who has some really nice pictures for this.)

Reply:
nextVersion.md just pushed.